### PR TITLE
tools/dm: add manage-task.md and precheck.md

### PR DESCRIPTION
**Pull request by yikeke**
Via docs: https://github.com/pingcap/docs/blob/master/tools/dm/manage-task.md
docs-cn reference: 
https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/task-handling/dmctl-manual.md
https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/task-handling/task-commands.md

PTAL. @lilin90 @GregoryIan @csuzhangxc

### CODE REVIEW

File: tools/dm/manage-task.md [hunk 1]
```diff
@@ -0,0 +1,595 @@
+---
+title: 管理数据同步任务
+category: tools
+---
+
+# 管理数据同步任务
+
+本文说明了如何使用 [dmctl](/tools/dm/overview.md#dmctl) 组件来进行数据同步任务的管理和维护。对于用 Ansible 部署的 DM 集群，dmctl 二进制文件路径为 `dm-ansible/dmctl`。
```
Comment (csuzhangxc) [inline tools/dm/manage-task.md]:
```suggestion
本文说明了如何使用 [dmctl](/tools/dm/overview.md#dmctl) 组件来进行数据同步任务的管理和维护。对于用 DM-Ansible 部署的 DM 集群，dmctl 二进制文件路径为 `dm-ansible/dmctl`。
```

File: tools/dm/manage-task.md [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+    - 指定在特定的一组 DM-workers 上执行 `task.yaml`
+    - 如果设置，则只启动指定任务在该组 DM-workers 上的子任务
++ `config_file`：
+    - 必选
+    - 指定 `task.yaml` 的文件路径
+
+#### 返回结果示例
+
+```bash
+{
```
Comment (csuzhangxc) [inline tools/dm/manage-task.md]: insert a line
```
» start-task task.yaml
```
before this line (`{`) to make format uniform?

File: tools/dm/manage-task.md [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+
+- `-w`：
+    - 可选
+    - 查询在指定的一组 DM-workers 上运行的数据同步任务的子任务
+- `task_name`：
+    - 可选 
+    - 指定任务名称 
+    - 如果未设置，则返回全部数据同步任务的查询结果
+
+#### 返回结果示例
```
Comment (csuzhangxc) [inline tools/dm/manage-task.md]: for `query-status`, we have a doc with comments for response fields, should we link to it?

File: tools/dm/manage-task.md [hunk 4]
```diff
@@ -0,0 +1,595 @@
...
+​            "op": "Pause",
+​            "result": true,
+​            "worker": "172.16.30.16:10081",
+​            "msg": ""
+​        }
+​    ]
+}
+```
+
+### 重启数据同步任务
```
Comment (csuzhangxc) [inline tools/dm/manage-task.md]: **resume** a **paused** task, **重启** or **恢复**, which is better?

File: tools/dm/manage-task.md [hunk 5]
```diff
@@ -0,0 +1,595 @@
...
+# 使用 `dmctl [command] --help` 来获取某个命令的更多信息
+```
+
+## 上游 MySQL 实例配置前置检查
+
+为提前检测出数据同步任务可能存在的一些配置错误，DM 提供了前置检查功能。`check-task` 命令可用于对上游 MySQL 实例配置是否满足 DM 要求进行前置检查。
+
+上下游数据库用户必须具备相应读写权限。当数据同步任务启动时，DM 会自动检查下列权限和配置：
+
++ 数据库版本
```
Comment (GregoryIan) [inline tools/dm/manage-task.md]: 我们可能会持续补充，而且这个内容很重要，我们可以单独把它放到一个文档中吗？

File: tools/dm/manage-task.md (hunk 4 above)
Comment (GregoryIan) [reply tools/dm/manage-task.md]: 恢复

**Comment by GregoryIan** [comment]
Rest LGTM

File: tools/dm/manage-task.md (hunk 5 above)
Comment (yikeke) [reply tools/dm/manage-task.md]: 好的，我把前置检查这部分放到单独一个文档中，就叫 precheck.md 可以吗？

Comment (yikeke) [reply tools/dm/manage-task.md]: Comment addressed. PTAL again. @GregoryIan

**Comment by GregoryIan** [comment]
LGTM

**Comment by csuzhangxc** [comment]
LGTM. @lilin90 PTAL

**Pull request merged**